crates/libs/core/src/handles.rs

```diff
@@ -0,0 +1,45 @@
+/// Custom code to free a handle.
+///
+/// This is similar to the `Drop` trait, and may be used to implement `Drop`, but allows handles
```

Comment:
Note that intradoc links are nice :)

```suggestion
/// This is similar to the [`Drop`] trait, and may be used to implement [`Drop`], but allows handles
```